crt_portal/cts_forms/templates/vot_resources.html

```diff
@@ -0,0 +1,111 @@
+{% extends "landing.html" %}
+
+{% load static %}
+{% load i18n %}
+
+{% block meta_title %}
+  {% trans "Civil Rights Division - Department of Justice - Voting resources" as meta_title %}
+  <meta property="og:title" content="{{meta_title}}" />
+  <meta property="name" content="{{meta_title}}" />
+{% endblock meta_title %}
+
+{% block page_title %}
+  <title>
+    {% trans "Civil Rights Division - Department of Justice - Voting resources" %}
+  </title>
+{% endblock %}
+
+{% block content %}
+<div class="crt-landing--section crt-landing--pale crt-subpage">
+  <div class="grid-container">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-4">
+        <ul class="usa-sidenav crt-subpage--nav" id="toc">
+          <li class="usa-sidenav__item">
+            <a href="#get-help">{% trans "How Do I..." %}</a>
```

Comment:
Should this be changed to "Report Voting Issues" to mirror the housing resources page?